lib/Controller/MessagesController.php

```diff
@@ -716,28 +733,40 @@ public function removeTag(int $id, string $imapLabel): JSONResponse {
 	 *
 	 * @param int $accountId
 	 * @param string $folderId
-	 * @param int $id
+	 * @param int[] $ids
 	 *
 	 * @return JSONResponse
 	 *
 	 * @throws ClientException
 	 * @throws ServiceException
 	 */
-	public function destroy(int $id): JSONResponse {
```

Comment:
the old `destroy` method might still be used as *messages* are registered as resource in routes.php. Could you restore the old method but `throw new \OCA\Mail\Exception\NotImplemented();`?

Reply:
done